Added Love in index.html

00aney opened the pull request:
Added Love in index.html

richie-south commented:
Looks like you only added you name to contributors.html 😕 Where is the love ❤️ ?

BennyCarlsson commented:
<3

00aney commented:
> Looks like you only added you name to contributors.html 😕 Where is the love ❤️ ?

Sorry, I removed it when I resolve the merge conflicts. I added it again
